modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -81,7 +81,7 @@ To replace the certificates, you can replace the installed CA certificate with t
 
 .Procedure: Replacing Existing Certificates
 
-. On the {productname} Server, at the command prompt, call the command `mgr-ssl-cert-setup` and provide the certificates as parameters:
+. On the {productname} container host, at the command prompt, call [command]``mgr-ssl-cert-setup`` and provide the certificates as parameters:
```

Comment:
I think we need a better wording here. `mgr-ssl-cert-setup` command does not exist on the "container host" but "inside of the container. Not so sure how to make this clear. Maybe 

```suggestion
. On the {productname} container host, at the command prompt, call [command]``mgr-ssl-cert-setup`` inside the container and provide the certificates as parameters:
```